registry/server/migrations/20201012152003_settings_meta.ts

```diff
@@ -0,0 +1,25 @@
+import * as Knex from "knex";
+import {
+    Scope,
+} from "../settings/interfaces";
+
+
+export async function up(knex: Knex): Promise<void> {
+    return knex.schema.table('settings', table => {
+        table.string('default', 50).notNullable().defaultTo(JSON.stringify(''));
```

Comment:
type for col `value` & `default` should match. So it must be TEXT here.

Reply:
Oh, I missed it, I looked at the `key`
I am going to change it